app/assets/javascripts/admin/utils/filters/unlocalize_currency.js.coffee

```diff
@@ -0,0 +1,8 @@
+angular.module("admin.utils").filter "unlocalizeCurrency", ()->
+  # Convert string to number using injected currency configuration.
+  (price) ->
+    # used decimal separator from currency configuration
+    decimal_separator = I18n.toCurrency(.1, {precision: 1, unit: ''}).substring(1,2)
+    if (decimal_separator == ",")
+      price = price.replace(",", ".")
+    return parseFloat(price, 10)
```

Comment:
Is this rounding to 10 decimal places? Why not just omit the second parameter?

Reply:
I was sure that `parseFloat` accept second parameters but this is only for `parseInt` (and this is a radix parameter)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseFloat